_posts/2015-09-17-modern-blog.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "ModernBlog"
+date:   2015-0917 01:01:00
```

Comment:
Please remove the extra whitespaces and quotes, also please fix typo in the date:

```
title: ModernBlog
date: 2015-09-17 01:01:00
```